spec/src/main/asciidoc/javase/bootstrap_se.asciidoc

```diff
@@ -101,6 +101,9 @@ If it is called and the container was already shutdown, it throws an `IllegalSta
 * `getBeanManager()` method returns the `BeanManager` (as defined in <<beanmanager>>) for the running container.
 If it is called and the container was already shutdown, it throws an `IllegalStateException`.
 
+When performing lookup of bean instance with `SeContainer` the qualifiers specified in the `Instance.select()` method are used to perform typesafe resolution.
```

Comment:
Like with `javax.enterprise.inject.spi.CDI` - this would deserve some general intro.